var/spack/repos/builtin/packages/gsl/package.py

```diff
@@ -29,7 +29,8 @@ class Gsl(AutotoolsPackage, GNUMirrorPackage):
     variant('external-cblas', default=False, description='Build against external blas')
 
     # from https://dev.gentoo.org/~mgorny/dist/gsl-2.3-cblas.patch.bz2
-    patch('gsl-2.3-cblas.patch', when="+external-cblas")
+    patch('gsl-2.3-cblas.patch', when="@:2.5 +external-cblas")
+    patch('gsl-2.6-cblas.patch', when="@2.6: +external-cblas")
```

Comment:
Did you create this patch yourself or does it come from somewhere?

Reply:
This is a copy of the `gsl-2.3-cblas.patch` file, with a change to the wavelet hunk.
```
diff -u var/spack/repos/builtin/packages/gsl/gsl-2.*.patch
--- var/spack/repos/builtin/packages/gsl/gsl-2.3-cblas.patch    2020-08-20 15:27:46.433233535 -0500
+++ var/spack/repos/builtin/packages/gsl/gsl-2.6-cblas.patch    2021-04-13 16:33:42.023213171 -0500
@@ -316,8 +316,8 @@
  
  TESTS = $(check_PROGRAMS)
  
--test_LDADD = libgslwavelet.la ../blas/libgslblas.la ../cblas/libgslcblas.la ../matrix/libgslmatrix.la ../vector/libgslvector.la ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la  ../err/libgslerr.la ../test/libgsltest.la ../sys/libgslsys.la ../utils/libutils.la
-+test_LDADD = libgslwavelet.la ../blas/libgslblas.la @CBLAS_LINK_LIBS@ ../matrix/libgslmatrix.la ../vector/libgslvector.la ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la  ../err/libgslerr.la ../test/libgsltest.la ../sys/libgslsys.la ../utils/libutils.la
+-test_LDADD = libgslwavelet.la ../matrix/libgslmatrix.la ../vector/libgslvector.la ../blas/libgslblas.la ../cblas/libgslcblas.la ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la  ../err/libgslerr.la ../test/libgsltest.la ../sys/libgslsys.la ../utils/libutils.la
++test_LDADD = libgslwavelet.la ../matrix/libgslmatrix.la ../vector/libgslvector.la ../blas/libgslblas.la @CBLAS_LINK_LIBS@ ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la  ../err/libgslerr.la ../test/libgsltest.la ../sys/libgslsys.la ../utils/libutils.la
  
  test_SOURCES = test.c
```
So, the source for me was the existing patch.

Reply:
@scheibelp Does this look okay to be merged?

Reply:
Yes - thanks and sorry for taking so long to check back on this!